src/VolleyManagement.Backend/VolleyManagement.Data.MsSql/VolleyManagement.Data.MsSql.csproj

```diff
@@ -251,6 +251,10 @@
     <Compile Include="Context\VolleyManagementEntities.cs" />
   </ItemGroup>
   <ItemGroup>
+    <ProjectReference Include="..\VolleyManagement.Contracts\VolleyManagement.Contracts.csproj">
```

Comment:
Why do you need this?

Reply:
My bad. Added by mistake during structure change. Will remove.